code/modules/mod/modules/modules_general.dm

```diff
@@ -633,3 +633,102 @@
 
 /obj/item/mod/module/plasma_stabilizer/on_unequip()
 	REMOVE_TRAIT(mod.wearer, TRAIT_NOSELFIGNITION, MOD_TRAIT)
+
+//Finally, https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real:
+///Hat Stabilizer - Allows displaying a hat over the MOD-helmet, à la plasmamen helmets.
+/obj/item/mod/module/hat_stabilizer
+	name = "MOD hat stabilizer module"
+	desc = "A simple set of deployable stands, directly atop one's head; these will deploy under a select few hats to keep them from falling off, allowing them to be worn atop the sealed helmet.\
+	\nYou still need to take the hat off your head while the helmet deploys, though.\
+	\nThis is a must-have for Nanotrasen Captains, enabling them to show off their authoritative hat even while in their MODsuit."
+	icon_state = "hat_holder"
+	module_type = MODULE_PASSIVE //The description says it auto-deploys for certain hats; that's just an excuse to have a whitelist of items.
+	complexity = 0 //Its FREE!
+	removable = TRUE //Inbuilt into the Captain's Magnate suit, and the CC Corporate suit; BUT also avaliable as a maint module.
```

Comment:
inherited from parent

Reply:
Left intentionally, as its a removable-but-0-complexity module by design, and I need to note as much in the code. Being able to steal the hat-wearing module is funny to me.

Maybe I can make it just a comment though... yeah, I'll just mention it in a comment